### DISCUSSION TITLE
Multiple tools with different widths

### DISCUSSION THREAD

**Discussion by GeorgePikoulis** [Courseplay/courseplay#7399] [Show and tell]
Hello guys,

Here's a little guide I wrote up in case anyone would like to use Courseplay with multiple tools of different widths.
I've found this practice useful because I don't like to sell starting machinery, but keep using it.
In my example I am using a 3m, an 8m and a 12m seeder.

- Create a course without using the multiple tools Courseplay feature, rather using the sum width; for this example 23m
- Open and copy [this google spreadsheet](https://docs.google.com/spreadsheets/d/1hU8SKKtHhKzdIfoW3G3FVQeL9Ara6MgN1KD7ZZuXGZA/edit?usp=sharing) for your own use
- Input the number of implements and start typing each width
- Remove or add rows by copying and pasting the offset formulae as required to create all the implements you need
- Go back in the game and copy the course to all participating vehicles
- Set the offset as calculated by the spreadsheet
- Preferably use the convoy mode

I've used this successfully with 3 headlands and simple up/down lanes.
No clue how other lane configurations would work. 

Thanks for your dedicated work with Courseplay :)

**Comment by Tensuko** [comment]
Symetic lange chang option might be important as well! I am not sure right now what I did the last time... I think I kept it as default.

**Reply by GeorgePikoulis** [reply]
I've left it as default too. Not sure if it does anything when not using the built-in multiple tools feature,
